src/idsk/components/skip-link/_skip-link.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../govuk/settings/all";
+@import "../../../govuk/tools/all";
+@import "../../../govuk/helpers/all";
+
+@include govuk-exports("idsk/component/skip-link") {
+  .govuk-skip-link {
```

Comment:
uz ked mame teraz kus viac casu a prenasame tie komponenty k nam tak prepisme aj tie classy uz na nase idsk-skip-link pretoze aj tak tam kde sa to pouziva vkladas nase macro "idskSkipLink"